<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Sensor Network</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_Start">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_nodenumbers">
             <property name="accessibleName">
              <string/>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="placeholderText">
              <string>select_number_of_nodes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_Status">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>Status Clear !!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_current_warnings">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>No warnings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Devices_available">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>300</height>
            </size>
           </property>
           <property name="text">
            <string>Devices: Availble Yes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Maschine Overview Page 1</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_node_1">
           <property name="text">
            <string>Node 1  </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_signal_node_1">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_1"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_change_signal_node_1">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_1">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_node_2">
           <property name="text">
            <string>Node 2   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_signal_node_2">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_change_signal_node_2">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_2">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_node_3">
           <property name="text">
            <string>Node 3   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_signal_node_3">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_change_signal_node_3">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_3">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_4"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Maschine Overview Page 2</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_node_4">
           <property name="text">
            <string>Node 4  </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_signal_node_4">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_6">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_node_5">
           <property name="text">
            <string>Node 5   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_signal_node_5">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_5"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_7">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_6"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_node_6">
           <property name="text">
            <string>Node 6   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="label_signal_node_6">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_6"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_12">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Record_node_8">
             <property name="text">
              <string>Start Recording</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_7"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Maschine Overview Page 3</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>861</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_node_7">
           <property name="text">
            <string>Node 7   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_7"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_14">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_15">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_8"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_node_8">
           <property name="text">
            <string>Node 8   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_8"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_16">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_17">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_9"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_node_9">
           <property name="text">
            <string>Node 9   </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_node_9"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_18">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_19">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_node_10"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
